views/account/login.pug

```diff
@@ -47,6 +47,9 @@ block content
         a.btn.btn-block.btn-github.btn-social(href='/auth/github')
           i.fab.fa-github.fa-sm
           | Sign in with GitHub
+        a.btn.btn-block.btn-imgur.btn-social(href='/auth/imgur')
+          i.fas.fa-image.fa-sm
+          | Sign in with Imgur
```

Comment:
We aren't doing sign in by imgur, because they aren't a big Oauth provider.
There are two places that someone will be able to link their web account to imgur:
- The message at the top of the API example page.
- At the very bottom of their account page.